src/components/Demo/buildEntities.tsx

```diff
@@ -75,13 +76,13 @@ export const AliceCreatesPolicy = ({ enabled, policyParams, setPolicyParams, gra
         />
       </FormRow>
       <FormRow>
-        <Button onClick={grantToBob}>Create policy</Button>
+        <Button onClick={grantToBob}>Build tDec Entities</Button>
```

Comment:
Careful with the onClick action `grantToBob` - or is it that we'll modify that once functionality is available?